src/main/java/com/griefcraft/model/Protection.java

```diff
@@ -1045,14 +1057,14 @@ public void sendProtectionInfo(CommandSender sender) {
                 .timeToString((System.currentTimeMillis() / 1000L)
                         - this.lastAccessed);
 
-        /*
-        TODO: Translate
-        if (!lastAccessed.equals("Not yet known")) {}
-        */
+        String lastAccessedLocale = lwc.getLocaleMessage(sender, "lwc.time.past", "time", lastAccessed)[0];
+        if (lastAccessed.equals("Not yet known")) {
```

Comment:
These should not need to be checked here. It's better coding style to return the translated string instead of the English one in TimeUtil::timeToString(long). That way this won't need to be re-translated in every method that uses it.